datafusion/src/physical_plan/planner.rs

```diff
@@ -506,6 +506,8 @@ impl DefaultPhysicalPlanner {
                     })
                     .collect::<Result<Vec<_>>>()?;
 
+                //It's not obvious here, but "order" here is mapping from input "sort_on" into
```

Comment:
The comment would fit better in the docstring of `compute_aggregation_strategy`. Would also help if it were to describe in which way `order` relates to `group by` column positions.